tests/api/test_api.py

```diff
@@ -48,10 +46,10 @@ def test_api_context(cript_api: cript.API) -> None:
     assert cript.api.api._get_global_cached_api() is not None
 
 
-def test_api_cript_env_vars() -> None:
+def test_create_api_with_none() -> None:
     """
-    tests that when the cript.API is given None for host, api_token, storage_token that it can correctly
-    retrieve things from the env variable
+    tests that when the cript.API is given `None` for `host`, `api_token`, `storage_token`
+    that it can correctly retrieve them from the environment variables
     """
     host_value = "http://development.api.mycriptapp.org/"
```

Comment:
Shouldn't we have those exported in the env anyways?
When you run the tests?

It can be confusing, if you set your env variables to connect to `staging` and then you run the tests and they fail because `development` is down.

Reply:
yes, but I thought this would be an easier way to write the tests so its modular and not dependent on the CI variables, so we can change the CI variables without this going off that it needs to be changed too. It is more self contained in the function, easier to read, and easier to write assertions for.

I can change it if we think getting it directly from the CI is better, it is not a huge deal to me.

Reply:
I think we as knowledgable engineers that run the tests should be comfortable with ENV variables.
I'd like it consistent with them

Reply:
> I think we as knowledgable engineers

I appreciate the vote of confidence, but let me tell you, I've made some of the dumbest mistakes imaginable before 😂 

but yeah I can change it to depend on the environment instead of being in the function